core-services/image-mirroring/openshift/mapping_origin_4_5

```diff
@@ -65,6 +65,7 @@ registry.svc.ci.openshift.org/origin/4.5:coredns quay.io/openshift/origin-coredn
 registry.svc.ci.openshift.org/origin/4.5:crd-schema-gen quay.io/openshift/origin-crd-schema-gen:4.5 quay.io/openshift/origin-crd-schema-gen:4.5.0 quay.io/openshift/origin-crd-schema-gen:latest
 registry.svc.ci.openshift.org/origin/4.5:csi-cluster-driver-registrar quay.io/openshift/origin-csi-cluster-driver-registrar:4.5 quay.io/openshift/origin-csi-cluster-driver-registrar:4.5.0 quay.io/openshift/origin-csi-cluster-driver-registrar:latest
 registry.svc.ci.openshift.org/origin/4.5:csi-driver-manila quay.io/openshift/origin-csi-driver-manila:4.5 quay.io/openshift/origin-csi-driver-manila:4.5.0 quay.io/openshift/origin-csi-driver-manila:latest
+registry.svc.ci.openshift.org/origin/4.5:csi-driver-manila-operator quay.io/openshift/origin-csi-driver-manila-operator:4.5 quay.io/openshift/origin-csi-driver-manila-operator:4.5.0
```

Comment:
You should not be adding these until the images are built at least once.

Reply:
done!